jumanji/wrappers_test.py

```diff
@@ -228,6 +235,25 @@ def test_jumanji_environment_to_deep_mind_env__unwrapped(
         """Validates unwrapped property of the wrapped environment."""
         assert isinstance(fake_dm_env.unwrapped, Environment)
 
+    def test_jumanji_environment_to_deep_mind_env__key_getter_setter(
+        self, fake_environment: FakeEnvironment)-> None:
+        """validate state getter and setter"""
+        key = jax.random.PRNGKey(0)
+        fake_dm_env = JumanjiToDMEnvWrapper(fake_environment, key=key)
+        assert jnp.array_equal(key, fake_dm_env.key)
+        new_key = jax.random.PRNGKey(10)
+        fake_dm_env.key = new_key
+        assert jnp.array_equal(new_key, fake_dm_env.key)
+    
+    def test_jumanji_environment_to_deep_mind_env__state_getter_setter(
+        self, fake_environment: FakeEnvironment, fake_state: FakeState
+        ) -> None:
+        """validate state getter and setter"""
+        key = jax.random.PRNGKey(0)
+        fake_dm_env = JumanjiToDMEnvWrapper(fake_environment, key=key)
+        fake_dm_env.state = fake_state
+        assert fake_state == fake_dm_env.state
```

Comment:
I think we even want to test reference equality here.
```suggestion
        assert fake_state is fake_dm_env.state
```